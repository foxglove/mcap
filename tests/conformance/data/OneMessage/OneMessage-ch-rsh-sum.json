{
  "meta": {
    "variant": {
      "features": [
        "ch",
        "rsh",
        "sum"
      ]
    }
  },
  "records": [
    {
      "fields": [
        [
          "library",
          ""
        ],
        [
          "profile",
          ""
        ]
      ],
      "type": "Header"
    },
    {
      "fields": [
        [
          "data",
          [
            "4",
            "5",
            "6"
          ]
        ],
        [
          "encoding",
          "c"
        ],
        [
          "id",
          "1"
        ],
        [
          "name",
          "Example"
        ]
      ],
      "type": "Schema"
    },
    {
      "fields": [
        [
          "id",
          "1"
        ],
        [
          "message_encoding",
          "a"
        ],
        [
          "metadata",
          {
            "foo": "bar"
          }
        ],
        [
          "schema_id",
          "1"
        ],
        [
          "topic",
          "example"
        ]
      ],
      "type": "Channel"
    },
    {
      "fields": [
        [
          "channel_id",
          "1"
        ],
        [
          "data",
          [
            "1",
            "2",
            "3"
          ]
        ],
        [
          "log_time",
          "2"
        ],
        [
          "publish_time",
          "1"
        ],
        [
          "sequence",
          "10"
        ]
      ],
      "type": "Message"
    },
    {
      "fields": [
        [
          "data_section_crc",
          "2475508533"
        ]
      ],
      "type": "DataEnd"
    },
    {
      "fields": [
        [
          "data",
          [
            "4",
            "5",
            "6"
          ]
        ],
        [
          "encoding",
          "c"
        ],
        [
          "id",
          "1"
        ],
        [
          "name",
          "Example"
        ]
      ],
      "type": "Schema"
    },
    {
      "fields": [
        [
          "group_length",
          "34"
        ],
        [
          "group_opcode",
          "3"
        ],
        [
          "group_start",
          "202"
        ]
      ],
      "type": "SummaryOffset"
    },
    {
      "fields": [
        [
          "summary_crc",
          "1380728824"
        ],
        [
          "summary_offset_start",
          "236"
        ],
        [
          "summary_start",
          "202"
        ]
      ],
      "type": "Footer"
    }
  ]
}
