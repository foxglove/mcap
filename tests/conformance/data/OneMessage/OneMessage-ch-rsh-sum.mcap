version https://git-lfs.github.com/spec/v1
oid sha256:853dc05be8c7dd1c9dd8bdbf4208e9c30b7133bdb74ca615850cdee04df94656
size 428
