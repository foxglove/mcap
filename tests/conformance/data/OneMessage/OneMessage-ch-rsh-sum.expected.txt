Header library=s:"" profile=s:""
Schema data=b:<040506> encoding=s:c id=i:1 name=s:Example
ChannelInfo id=i:1 message_encoding=s:a metadata=m:{s:foo=s:bar} schema_id=i:1 topic=s:example
Message channel_id=i:1 data=b:<010203> log_time=n:2 publish_time=n:1 sequence=i:10
Schema data=b:<040506> encoding=s:c id=i:1 name=s:Example
ChunkIndex chunk_length=n:162 chunk_start_offset=n:25 compressed_size=n:115 compression=s:"" end_time=n:2 message_index_length=n:0 message_index_offsets=m:{} start_time=n:2 uncompressed_size=n:115
SummaryOffset group_length=n:34 group_opcode=i:3 group_start=n:202
SummaryOffset group_length=n:0 group_opcode=i:13 group_start=n:236
SummaryOffset group_length=n:77 group_opcode=i:8 group_start=n:236
Footer summary_crc=i:0 summary_offset_start=n:313 summary_start=n:202
